documentation/assemblies/assembly-iot-guide.adoc

```diff
@@ -11,23 +11,10 @@ The IoT guide provides resources on how to set up and manage {ProductName} IoT f
 
```

Comment:
I don't think this assembly is needed any longer, since the information will be split between the getting started, service admin, and tenant admin sections (eventually books)